cmd/spire-server/cli/entry/util.go

```diff
@@ -91,6 +95,34 @@ func printEntry(e *common.RegistrationEntry) {
 	fmt.Println()
 }
 
+// ParseFile parses JSON represented RegistrationEntries
+// if path is "" or "*" read JSON from STDIN
```

Comment:
Replace `"*"` with `"-"`

Also, I'm not confident we want to alias an empty path as stdin.. as it could mask user error.

Reply:
Yes, the empty path doesn't work well.
The comment seems to have just been forgotten to remove.